big_tests/tests/accounts_SUITE.erl

```diff
@@ -386,4 +386,5 @@ watcher(Watcher) ->
 domain() ->
     ct:get_config({hosts, mim, domain}).
 
-			
+make_jid(U, S) ->
+    rpc(mim(), jid, make, [U, S, <<>>]).
```

Comment:
I think we have a helper function for this in `mongoose_helper` module in big tests.

Reply:
Moving the `make_jid/2` to the `mongoose_helper` may be handy in other places.